backend/api/utils.py

```diff
@@ -0,0 +1,10 @@
+import json
+from bson import json_util
+from typing import Any
+
+from rest_framework.response import Response
+
+
+class ApiResponse(Response):
```

Comment:
I believe the builtin `Response` in DRF can already serialize and return json objects.

Reply:
It didn't do the work for me. I found that it's because DRF `Response` was not able to serialize `Cursor` objects returned by the `.find()` method of `Collections`.
(for example as used in `api/views/quiz_view.py`, line 10)

That's why I use `json_utils` in the `bson` package.

Reply:
Okay.! So you cannot use json.dumps also ?

Reply:
Nope

Reply:
Okay 😉